src/cli.py

```diff
@@ -6,14 +6,25 @@
 
 class CmdApp(Cmd):
 
-    Cmd.shortcuts.update({'!': 'bang', '$': 'shell'})
+    def __init__(self):
+        Cmd.__init__(self)
+        Cmd.shortcuts.update({'!': 'bang', '$': 'shell'})
+        self.abbrev = True
+        self.prompt = '[] > '
 
     def precmd(self, line):
         if not line.startswith('load') and \
            (line.endswith('.h5') or line.endswith('.hdf5')):
             line = 'load ' + line
         return line
 
+    def postcmd(self, stop, line):
+        if self.explorer:
+            self.prompt = '[{}: {}/] > '.format(
```

Comment:
It is simply the basename of the directory you are currently in

my Bash prompt, for the record, is similar [user@host directory]$, not shown relative to ~ or /, just the name of the directory